File: library/src/main/java/com/raxdenstudios/square/activity/interceptor/AutoInflateLayoutInterceptor.java
package com.raxdenstudios.square.activity.interceptor;

import android.os.Bundle;
import android.view.View;

import com.raxdenstudios.square.Interceptor;
import com.raxdenstudios.square.activity.interceptor.delegate.AutoInflateLayoutInterceptorDelegate;

/**
 * Created by agomez on 22/05/2015.
 */
public interface AutoInflateLayoutInterceptor extends Interceptor {

    void onInterceptorCreated(AutoInflateLayoutInterceptorDelegate callback);

    void onViewCreated(View view, Bundle savedInstanceState);

}

File: square-commons/src/main/java/com/raxdenstudios/square/interceptor/commons/autoinflatelayout/AutoInflateLayoutInterceptor.kt
package com.raxdenstudios.square.interceptor.commons.autoinflatelayout

import com.raxdenstudios.square.interceptor.Interceptor

/**
 * Created by Ángel Gómez on 22/05/2015.
 */
interface AutoInflateLayoutInterceptor : Interceptor {

    fun setLayoutId(layoutId: Int)
}
